sig-cluster-lifecycle/charter.md

```diff
@@ -0,0 +1,65 @@
+# SIG Cluster Lifecycle Charter
+
+## Scope
+
+SIG Cluster Lifecycle’s objective is to improve the deployment, configuration management, and upgrade story for Kubernetes clusters and relevant infrastructure.
+
+### In scope
+
+The following topics fall under ownership of this SIG:
+
+- Improving the Kubernetes UX for cluster administration
```

Comment:
should we say "user experience" instead of UX to avoid jargon?

Reply:
It isn't just about automation, it's about making a good user experience for the touch points of the cluster admin persona. Automation is one way to make a good user experience, but it isn't necessarily the correct approach for all user journeys.

Reply:
Fair enough. 

As you can probably tell from a number of my comments, I'm struggling with seeing the bounds of the SIG from the text that's in the charter. I think there is a narrower scope in practice. I'm trying to reverse engineer what it is.